examples/ios/PickerView.swift

```diff
@@ -0,0 +1,267 @@
+//
+//  PickerView.swift
+//  React Native Segmented Picker
+//
+//  Created by Adam McArthur on 20/6/20.
+//
+
+import UIKit
+
+class PickerView: UIView, UIPickerViewDelegate, UIPickerViewDataSource {
+  let picker: UIPickerView = UIPickerView()
+  var options = [PickerColumn]()
+  var defaultSelectionsApplied: Bool = false
+  var theme: PickerTheme = PickerTheme(theme: nil)
+
+  // React Props
+  @objc(options) var optionsProp: [NSDictionary] = [] {
+    didSet {
+      parseOptionsProp()
+    }
+  }
+  @objc(defaultSelections) var defaultSelectionsProp = [String: String]() {
+    didSet {
+      applyDefaultSelections()
+    }
+  }
+  @objc(theme) var themeProp = NSDictionary() {
+    didSet {
+      theme = PickerTheme(theme: themeProp)
+    }
+  }
+  @objc var onValueChange: RCTBubblingEventBlock?
+  @objc var onEmitSelections: RCTBubblingEventBlock?
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    picker.dataSource = self
+    picker.delegate = self
+    self.addSubview(picker)
+  }
+
+  /**
+   * Setup the picker frame when the view appears on the screen.
+   */
+  override func willMove(toWindow newWindow: UIWindow?) -> Void {
+    super.willMove(toWindow: newWindow)
+    // AKA: View Did Appear
+    if newWindow != nil {
+      picker.frame = self.frame
+      let selectionHighlighter = UIView(frame: CGRect(
+        x: 0,
+        y: 0,
```

Comment:
You could get rid of line 63-65 and set the y: to `(picker.frame.height-theme.itemHeight)/2`